variables.tf

```diff
@@ -307,12 +307,9 @@ DESCRIPTION
   nullable    = false
 
   validation {
```

Comment:
this is the only change required for the feature. The versioning changes are related to errors for `tflint`